vllm/worker/model_runner.py

```diff
@@ -21,16 +21,23 @@
 from vllm.lora.worker_manager import LRUCacheWorkerLoRAManager
 from vllm.lora.layers import LoRAMapping
 from vllm.lora.request import LoRARequest
-from vllm.utils import in_wsl, measure_cuda_memory
+from vllm.utils import in_wsl, measure_cuda_memory, _get_aligned_size
 
 logger = init_logger(__name__)
 
 KVCache = Tuple[torch.Tensor, torch.Tensor]
 _PAD_SLOT_ID = -1
 LORA_WARMUP_RANK = 8
-# Capture graphs for batch size 1, 2, 4, 8, 16, 24, 32, 40, ..., 256.
+_BATCH_SIZE_ALIGNMENT = 8
+# Capture graphs for token size 1, 2, 4, 8, 16, 24, 32, 40, ..., 256.
 # NOTE: _get_graph_batch_size needs to be updated if this list is changed.
-_BATCH_SIZES_TO_CAPTURE = [1, 2, 4] + [8 * i for i in range(1, 33)]
+_BATCH_SIZES_TO_CAPTURE = [1, 2, 4] + [
+    _BATCH_SIZE_ALIGNMENT * i for i in range(1, 33)
+]
+# True if inputs should be aligned. It is currently disabled.
+# Aligning inputs can better utilize tensor cores.
+# https://developer.nvidia.com/blog/optimizing-gpu-performance-tensor-cores/
+SHOULD_ALIGN = False
```

Comment:
```suggestion
_SHOULD_ALIGN = False
```

Reply:
nit: I personally feel we should either always do padding or always don't, for simplicity.

Reply:
Thanks for the benchmark!